packages/eslint-plugin/src/rules/keyword-spacing.ts

```diff
@@ -0,0 +1,598 @@
+import keywords from 'eslint/lib/rules/utils/keywords';
```

Comment:
Here I basically took `keyword-spacing.js` from `eslint` and tried to adapt it by watching how it was done for other existing rules.